rpcs3/rpcs3qt/emu_settings.cpp

```diff
@@ -217,6 +218,8 @@ emu_settings::Render_Creator::Render_Creator()
 		supportsVulkan = device_found;
 		vulkanAdapters = std::move(compatible_gpus);
 	}
+
+	enum_thread.join();
```

Comment:
Wouldn't this lock up the emulator if user selected ignore and the thread never finishes?